chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -884,6 +908,17 @@ void BraveRenderViewContextMenu::InitMenu() {
 #if BUILDFLAG(ENABLE_AI_CHAT)
   BuildAIChatMenu();
 #endif
+
+  // Add Open Link in Split View
+  if (CanOpenSplitViewForWebContents(source_web_contents_->GetWeakPtr()) &&
+      params_.link_url.is_valid()) {
+    index = menu_model_.GetIndexOfCommandId(IDC_CONTENT_CONTEXT_OPENLINKNEWTAB);
+    DCHECK(index.has_value());
```

Comment:
prefer CHECK

Reply:
In this file, all the index checks are DCHECKs. If you suggest using CHECK here, should I also change the others?

Reply:
let's just write new code with CHECK, as appropriate

Reply:
Sure, if you don't mind, could you please let me know why you feel it is appropriate to use CHECK instead of DCHECK here?

Also, I noticed that in some files, index validation is done via an if condition, like `if(index.has_value()) ...`, instead of using CHECKs. This approach avoids causing an intentional crash for end users (In my opinion,
having a bug would be a preferable alternative to a crash in this case). Correct me if I'm wrong.

And I have a concern about whether we can use both `DCHECK` (which helps to identify issues during the Dev/QA phase) and `if` conditions in these kinds of situations?

Reply:
see https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md#check_dcheck_notreached_noreturn_and-notreached_in_migration

Reply:
> see https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md#check_dcheck_notreached_noreturn_and-notreached_in_migration

Thanks for sharing @iefremov